app/views/layouts/_branding.html.erb

```diff
@@ -22,21 +22,59 @@
     </div>
 
     <!-- Collect the nav links, forms, and other content for toggling -->
-    <div class="collapse navbar-collapse" id="org-navbar-menu">
-      <% if user_signed_in? && !current_user.org.nil? && current_user.org.links.present? %>
-        <% if current_user.org.links['org'].present? %>
-            <ul class="nav navbar-nav">
-              <% current_user.org.links['org'].each do |url| %>
-                <% unless url['link'].blank? %>
-                  <li><%= link_to (url['text'].blank? ? url['link'] : url['text']), url['link'] %>
-                <% end %>
+
+      <!-- Navigation for organisation admin -->
+      <% if user_signed_in? && !current_user.org.nil? %>
+        <ul class="nav navbar-nav">
+          <li class="org-links">
+            <% if !current_user.org.links['org'][0].blank? %>
+              <%= link_to current_user.org.links['org'][0]['link'], :target=>'_blank', :class => 'org-a' do %>
+                <i class="fa fa-globe" aria-hidden="true">&nbsp;</i>
+                <%=(current_user.org.links['org'][0]['text'].blank? ? current_user.org.links['org'][0]['link'] : current_user.org.links['org'][0]['text'])%>
               <% end %>
-            </ul>
-        <% end %>
+            <% else %>
+              <%= mail_to current_user.org.contact_email, :class => 'org-a' do %>
+                <i class="fa fa-envelope" aria-hidden="true">&nbsp;</i>
+                <%= current_user.org.contact_name %>
+              <% end %>
+            <% end %>
+            <% if !current_user.org.links['org'][1].blank? %>
+              <%= link_to current_user.org.links['org'][1]['link'], :target=>'_blank', :class => 'org-a' do %>
+                <i class="fa fa-globe" aria-hidden="true">&nbsp;</i>
+                <%=(current_user.org.links['org'][1]['text'].blank? ? current_user.org.links['org'][1]['link'] : current_user.org.links['org'][1]['text'])%>
+              <% end %>
+            <% else %>
+              <%= mail_to current_user.org.contact_email, :class => 'org-a' do %>
```

Comment:
why are we showing the main contact email anytime one of the 3 possible links is missing

Reply:
ah, you're right @briri, that slipped through. Fixing and re-pushing.